markdown/docs/addon-author-guide.md

```diff
@@ -0,0 +1,321 @@
+# Addon Author Guide
+
+If you're the author of an Ember addon, you might need to make some
+changes to ensure that your addon runs correctly in the FastBoot
+environment.
+
+Remember that when updating your addons for FastBoot compatibility,
+your code now must run in both a browser *and* Node.js. Most
+importantly, that means that you cannot just assume that there will
+always be DOM APIs available.
+
+## Don't Break the Boot
+
+The first step towards FastBoot support is ensuring that your addon
+doesn't cause the app to crash when run in Node. You can verify this by
```

Comment:
@locks Is there a standard for this? I usually start by using the full name (Ember.js or Node.js) and subsequent uses I drop the .js (Ember/Node).